### DISCUSSION TITLE
Is it possible to delete all store data at once?

### DISCUSSION THREAD

**Discussion by marcusamargi** [CodeDredd/pinia-orm#1873] [General]
In Vuex-ORM you could delete all store data at once using

    store.dispatch('entities/deleteAll')

https://vuex-orm.org/guide/data/deleting.html#delete-all-data

I expect this can't be done with Pinia-ORM since there is no central place to register all the models. Is this possible?

Thank you

**Comment by CodeDredd** [comment]
You are right. this can't be done, because pinia-orm uses `composables`. The stores are registered on use, not on plugin init.
But you can easly write an composable `deleteAllPiniaOrmData` where you import each model and then call `delete` (with hooks)
or `flush`.

**Reply by marcusamargi** [reply]
thank you!
